### ISSUE TITLE
[MU4 Issue] When attempting to open a nonexistent score MU4 opens a new window

### ISSUE THREAD

**Issue by Trainzack** [musescore/MuseScore#8884]
**Describe the bug**
When attempting opening a score from "File > Open Recent" that doesn't exist, MU4 opens a new blank main application window anyway.

Note that this is different from #8875 because there the new window has the home tab displayed, whereas here the new window has no tabs.

**To Reproduce**
1. Create a new score
2. Create a second new score (this will open in a new window)
2. Save the first score
3. Close the MuseScore window that has the first score, keeping the second window open
4. Delete the score that was saved
5. From the remaining window, attempt to open the deleted score from the File > Open Recent menu
6. Click "ok" on the error dialogue that appears
7. Observe as a new, completely blank musescore window is opened.

**Expected behavior**
After clicking on the ok button during step 7, no new window should appear.

<details>
<summary>Screenshots</summary>

Warning message displayed as expected:
![image](https://user-images.githubusercontent.com/19575360/130169504-dc2475d6-6887-485b-aa97-12505f164a43.png)
Blank MuseScore main application window is unexpected:
![image](https://user-images.githubusercontent.com/19575360/130169589-1e2a1e44-04f2-4808-807c-39bca56aeddc.png)
</details>

**Desktop (please complete the following information):**
 - OS: Windows 10 Version 2009, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-, revision: github-musescore-musescore-
 - Also occurs on Mac12

**Comment by wizofaus** [comment]
When I tried I got

![image](https://user-images.githubusercontent.com/5364225/130176597-21a502f3-0fb1-46af-bd83-803166473c1d.png)

But I didn't follow your instructions exactly...

**Comment by wizofaus** [comment]
I also just had a scenario where after closing *just* the score window (but leaving the main window open), and switching to another window, the main musescore app window just disappeared! (the process exited). Wasn't running under the debugger so I have no idea why.

**Comment by Trainzack** [comment]
> Which is wrong, but no new window after dismissing.

I thought that dialog was intentional. What about that exactly is incorrect? 

> How did you bring up the second musescore window?

I had it open already. Now that I think about it, my instructions are flawed in that they don't specifically say how to get two different Musescore windows in the first place, and that might be important for reproduction. I'll have to revise them when I get back at it tomorrow.

**Comment by wizofaus** [comment]
Well it doesn't tell you *why* it couldn't open the file for a start, there's nothing after the :

**Comment by DmitryArefiev** [comment]
Tested on latest master. The behavior has changed (probably after #9915)

MU4 still opens a new blank window but closing it after click OK

https://user-images.githubusercontent.com/90187801/143886569-0dcad356-3d57-48f5-b331-4719cdb3349e.mp4

**Comment by DmitryArefiev** [comment]
And no longer occurs on Mac and Linux. Changing priority to P3

**Comment by HemantAntony** [comment]
Update, now it shows up like this

https://user-images.githubusercontent.com/88136054/156878307-b943bcab-a87f-412b-b8f5-70a7f3fdef90.mp4

This is the expected behavior, right?

**Comment by Trainzack** [comment]
@HemantAntony There is still a new, empty Musescore window being opened. I'm not sure that that's desirable in this situation.

**Comment by Tantacrul** [comment]
Don't think this requires fixing particularly.

**Comment by wizofaus** [comment]
I just tried again and it's still wrong - if I create/open a score, then try to open a non-existing file from recent files, I get a brand new top level window, then this dialog:

![image](https://user-images.githubusercontent.com/5364225/172505301-aa5bdd7b-7abd-48c0-a88d-fddfa926f248.png)

Not a showstopper, but it's not very nice.

**Comment by wizofaus** [comment]
Sorry, that wasn't the latest version, it now shows just this (but still opens a new top-level window first):

![image](https://user-images.githubusercontent.com/5364225/172607738-f0a6bc47-d6f5-445a-9e8c-74fda469c65e.png)

**Comment by DmitryArefiev** [comment]
> Sorry, that wasn't the latest version, it now shows just this (but still opens a new top-level window first):
> 
> ![image](https://user-images.githubusercontent.com/5364225/172607738-f0a6bc47-d6f5-445a-9e8c-74fda469c65e.png)

@Tantacrul I think we can fix it with editing text to "File not found". That scenario might happen when user just connected off his flash drive with the score recently saved. And probably that for 4.x.x. What do you think?

**Comment by DmitryArefiev** [comment]
> @DmitryArefiev yes, the most lightweight solution is best for now. Anything else bumped to 4.x


@Tantacrul @Trainzack I created a separate issue for File open error message (P3, Beta2) and want to put the current issue to 4.x.x version

**Comment by Tantacrul** [comment]
Cool!

**Comment by HemantAntony** [comment]
If this issue is closed, shouldn't it be removed from 'To do' in Multi_Project_System project?
